### PR TITLE
Run one iteration of performance tests in Outerloop runs

### PR DESCRIPTION
**Pull request by mellinoe**
This change will cause us to run one iteration of performance tests during our Outerloop runs. This will give us more assurance that the performance tests are not broken by various changes and will guarantee that they run, at least once.

@danmosemsft @DrewScoggins

### CODE REVIEW

File: Tools-Override/tests.targets [hunk 1]
```diff
@@ -51,7 +51,7 @@
     <XunitOptions Condition="'$(BuildingNETFxVertical)' == 'true' and '$(XUnitNoAppdomain)' == 'true'">$(XunitOptions) -noappdomain </XunitOptions>
     <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
 
-    <XunitOptions Condition="'$(Performance)'!='true'">$(XunitOptions) -notrait Benchmark=true</XunitOptions>
+    <XunitOptions Condition="'$(Performance)'!='true' And '$(Outerloop)' != 'true'">$(XunitOptions) -notrait Benchmark=true</XunitOptions>
```
Comment (danmosemsft) [inline Tools-Override/tests.targets]: nit, lowercase and

**Comment by danmosemsft** [comment]
Tizen is a known break, it is ok to ignore it.

**Comment by danmosemsft** [comment]
@dotnet-bot test innerloop Ubuntu16.04 arm Debug Cross Build (network issue)

**Comment by danmosemsft** [comment]
@dotnet-bot test Innerloop Tizen armel Debug Cross Build  (dirty enlistment)
@dotnet-bot test Innerloop Tizen armel Release Cross Build  (dirty enlistment)

**Comment by stephentoub** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netcoreapp Ubuntu14.04 Debug
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug

**Comment by stephentoub** [comment]
@mellinoe, looks like the System.IO.Pipes.Performance.Tests are hanging (or at least taking too long) on Unix.

**Comment by karelz** [comment]
Nice!

**Comment by mellinoe** [comment]
@stephentoub Yeah, it is hung on my Ubuntu machine. There are a couple of threads waiting on a PipeStream write and a socket event loop, so there seems to be some sort of deadlock in the test. I'll disable on Unix.

**Comment by mellinoe** [comment]
@dotnet-bot test outerloop netcoreapp Windows_NT Debug
@dotnet-bot test outerloop netcoreapp Ubuntu14.04 Debug
@dotnet-bot test outerloop netcoreapp OSX10.12 Debug

**Comment by stephentoub** [comment]
@dotnet-bot test Innerloop Windows_NT Debug x86 Build and Test please

**Pull request merged**